scripts/sync-docs.sh

```
@@ -0,0 +1,88 @@
+# --- clone target into a temp dir ---
+TMPDIR="$(mktemp -d)"
+trap 'rm -rf "$TMPDIR"' EXIT
+
+echo "Syncing docs → ${TARGET_REPO} (${TARGET_BRANCH})..."
+gh repo clone "$TARGET_REPO" "$TMPDIR/docs-site" -- --branch "$TARGET_BRANCH" --single-branch --depth 1 -q
+
+TARGET="$TMPDIR/docs-site/${TARGET_DIR}"
+
+# --- sync ---
+for dir in "${SYNC_DIRS[@]}"; do
+  rm -rf "${TARGET}/${dir}"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)